README.md

```diff
@@ -12,60 +12,60 @@ platform.
+- Q: Then that's cool. But I noticed that you are using [Go](http://golang.org) programming language. Do I need to install [Go](http://golang.org) compiler and other stuff to run `uniqush-push`?
+- A: No. There are no installation dependencies. All you need to do is to download the
   binary file from the [download page](http://uniqush.org/downloads.html) and
-install it. You only need to have a [redis](http://redis.io) database running
-somewhere so that *uniqush-push* can store the user data in
-[redis](http://redis.io). For more details, here is the [installation guide](http://uniqush.org/documentation/install.html)
+install it. But you do need to set up a [Redis](http://redis.io) server running
+somewhere, preferably with persistence, so that `uniqush-push` can store the
+user data in [Redis](http://redis.io). For more details, here is the
+[installation guide](http://uniqush.org/documentation/install.html)
 
-- Q: This is nice. I want to give it a try. But you are keep talking about *uniqush-push*, and I'm talking about *uniqush*, are they the same thing?
+- Q: This is nice. I want to give it a try. But you are keep talking about `uniqush-push`, and I'm talking about *Uniqush*, are they the same thing?
 - A: Thank you for your support! *Uniqush* is intended to be the name of a
   system which provides a full stack solution for communication between mobile
-devices and the app's server. *uniqush-push* is one piece of the system.
-However, right now, *uniqush-push* is the only piece and others are under
-active devlopment. If you want to know more details about the *uniqush*
+devices and the app's server. `uniqush-push` is one piece of the system.
+However, right now, `uniqush-push` is the only piece and others are under
+active development. If you want to know more details about the *Uniqush*
 system's plan, you can read the [blog
-post](http://blog.uniqush.org/uniqush-after-go1.html). If you want to know the
-latest progress about *uniqush* system, please check [our
-blog](http://blog.uniqush.org/). And if you are really impatient, [our github
-account](http://github.com/uniqush) will always contain the latest progress.
+post](http://blog.uniqush.org/uniqush-after-go1.html). If you want to find out
+about the latest progress with *Uniqush*, please check out [our
+ blog](http://blog.uniqush.org/). And, if you are really impatient, there's
+ always our [our GitHub account](http://github.com/uniqush) which could have
+ brand new stuff that hasn't been released yet.
 
 # Related Links #
 - [This story](http://uniqush.org/documentation/intro.html) may help you to understand
-the basic idea of *uniqush*.
+the basic idea of *Uniqush*.
 - [Documentation](http://uniqush.org/documentation/index.html)
-- [Uniqush Blog](http://blog.uniqush.org) announces the latest news about uniqush.
-
+- [The Uniqush blog](http://blog.uniqush.org) announces the latest news about Uniqush.
+- [Redis Persistence](http://redis.io/topics/persistence)
```

Comment:
This web page doesn't describe how one would configure a server to have persistence (or what to look at to check if a server has persistence).

I assume that the self-documenting configs linked in http://redis.io/topics/config , in the section `**PERSISTENCE**` of redis.conf, would be an okay example of what to do. (Not completely sure)

"Redis Persistence" -> "Redis persistence"

"Redis persistence describes the details of how redis saves data on shutdown, as well as how one might back up that data. Make sure that the redis server you use has persistence enabled - your redis.conf should have contents similar to the section `**PERSISTENCE**` of redis.conf in the example config files linked in http://redis.io/topics/config "